pom.xml

```diff
@@ -340,6 +340,34 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+                <plugin>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
+                    <!-- This is the last version with Java 8 support -->
+                    <version>2.30.0</version>
+                    <configuration>
+                        <!-- limit format enforcement to just the files changed by this feature branch -->
+                        <ratchetFrom>origin/master</ratchetFrom>
+                        <formats>
+                            <format>
+                                <includes>
+                                    <include>.gitattributes</include>
+                                    <include>.gitignore</include>
+                                </includes>
+                                <trimTrailingWhitespace/>
+                                <endWithNewline/>
+                                <indent>
+                                    <tabs>true</tabs>

```

Comment:
This means tabs are retained, but this section doesn't cover java code (that's handled in the `<java>` section below) but instead just `.gitattributes` and `.gitignore`. 

I think I'll just remove this section for now since the  focus of this change should just be the java source files, which is already a large enough change